src/hardware/ps1audio.h

```diff
@@ -0,0 +1,120 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2021-2024  The DOSBox Staging Team
+ *  Copyright (C) 2002-2021  The DOSBox Team
```

Comment:
Also suggest not indicating that 'The DOSBox Team' be credited as the author of this header because I'm looking in the SVN source, and it doesn't even break out PS1 audio, instead it's all part of the tandy sound code and they don't have things like (fifo, `MixerChannelPtr`, frame_counter, per-port IO functions, etc). So this header appears to be 100% Staging team.

Reply:
This is the same as what's in `ps1audio.cpp`.  I don't really want to change attributions willy-nilly.  I'm not really a fan of these headers in general for what it's worth.  I usually just throw this stuff in one file rather than bloating every single source-code file with this stuff.

Reply:
@weirddan455 I tend to agree, IMO it's overkill. The whole project is under the same license anyway, so it's pointless to copy the license text into in every single file.

The Duckstation guy only does this, which is sufficient IMO:

```
// SPDX-FileCopyrightText: 2019-2023 Connor McLaughlin <stenzek@gmail.com> and contributors.
// SPDX-License-Identifier: (GPL-3.0 OR CC-BY-NC-ND-4.0)
```

I'm happy to convert our headers to this condensed format:

```
// SPDX-License-Identifier: GPL-2.0-or-later
// Copyright (C) 2002-2024  The DOSBox and DOSBox Staging Teams
```

Then people can check out the commit history for specifics. Seems rather pointless to be super-exact about this stuff in the header files.

Reply:
Pushed a commit to change copyright to `Copyright (C) 2021-2024  The DOSBox Staging Team` (removing original DOSBox copyright attribution).

Up to you what you want to do with our overall formatting of these.  I really don't care enough to change it.  It's just a minor annoyance to keep track of this license stuff.  A condensed format would still have the same over-all problem of tracking attributions per-file.